src/app/static/src/js/components/reportLog/reportLog.vue

```diff
@@ -4,29 +4,28 @@
         <div v-if="showReports" id="logs-form-group" class="form-group row">
             <label for="report" class="col-sm-2 col-form-label text-right">Show logs for</label>
             <div class="col-sm-6">
-                <report-list id="logs" 
+                <running-reports-list id="logs" 
                 :reports="reports" 
-                :report.sync="selectedReport"
                 v-on="$listeners"
-                :key.sync="selectedLogReportKey"/>
+                :key.sync="selectedRunningReportKey"/>
```

Comment:
Is the intention that the report details component is going to go inside this one, or that it will be contained in the parent for this component? 
If the former, I think selectedRunningReportKey needs to be a prop rather than data, because it may be set externally to this, from the link from the run report tab, then this component will need to pass that value down to the detail component. In that case, it should emit the value and let the container update the prop. 
If the latter, this will still need to emit the value so that the parent can handle it.

Reply:
@EmmaLRussell @carbowizzy The structure I imagined was `runReportTabs` contains `runReport` and `reportLog`; `reportLog` contains `runningReportsList` and `runningReportDetails`. `runningReportsList` emits `selectedRunningReportKey` to `reportLog`, which is then passed as a prop to `runningReportDetails`. Would that not work?

Reply:
Yeah, that should work - so that's the 'former case' above. Just need to be aware that the selected running report is also going to be coming from the link in the run report page, so I guess that needs to come into reportLog as a prop via runReportTabs